cypress/integration/login/login_page.js

```diff
@@ -0,0 +1,39 @@
+import LoginPage from '../../support/page-objects/LoginPage'
+
+describe('login page elements', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/login')
+    cy.wrap(LoginPage()).as('page')
+  })
+  
+  it('displays user name text field', () => {
+    cy.get('@page').then((page) => {
+      page.username_input().should('be.visible')
+    })
+  })
+  
+  it('displays password text field', () => {
+    cy.get('@page').then((page) => {
+      page.password_input().should('be.visible')
+    })
+  })
+
+  it('displays submit button', () => {
+    cy.get('@page').then((page) => {
+      page.submit_button().should('exist')
+    })
+  })
+  
+  
+  describe('login in low', () => {
+    it('should let me login after entering username and password', () => {
```

Comment:
Is there any assertation on this test after the submit button click?

Reply:
Not as of now. I don't have the requirement to build the assertion. Wasn't sure if there's more work coming up with the demo branch.